include/thingset/can.h

```diff
@@ -73,18 +89,39 @@ extern "C" {
 #define THINGSET_CAN_TARGET_GET(id) \
     (((uint32_t)id & THINGSET_CAN_TARGET_MASK) >> THINGSET_CAN_TARGET_POS)
 
+#define THINGSET_CAN_ADDR_MIN       (0x01)
 #define THINGSET_CAN_ADDR_MAX       (0xFD)
 #define THINGSET_CAN_ADDR_ANONYMOUS (0xFE)
 #define THINGSET_CAN_ADDR_BROADCAST (0xFF)
 
-/* data IDs for publication messages */
+/* data IDs for single-frame reports */
```

Comment:
I think this should probably not be called a "report", as we are using "report" to refer to `0x1`-type broadcasts.

Reply:
Both of them are still reports in the ThingSet sense, they just endcode the endpoint and the payload differently.